gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -192,7 +193,7 @@ public Boolean load(String bucketName) throws Exception {
   @VisibleForTesting Storage storage;
 
   // Utility for building and caching storage channels and stubs.
-  private StorageStubProvider storageStubProvider;
+  @VisibleForTesting StorageStubProvider storageStubProvider;
```

Comment:
How about we keep the instance variable `private` and add a `package-private` getter method ?